ledger/internal/applications.go

```diff
@@ -54,6 +54,14 @@ type cowForLogicLedger interface {
 	blockHdrCached(round basics.Round) (bookkeeping.BlockHeader, error)
 }
 
+func (cs *roundCowState) AccountData(addr basics.Address) (ledgercore.AccountData, error) {
+	record, err := cs.Get(addr, true)
+	if err != nil {
+		return ledgercore.AccountData{}, err
+	}
+	return record, nil
+}
+
```

Comment:
@jannotti Sanity checking:  `AccountData` appears to duplicate a pre-existing definition several lines below.  Can the PR's addition be removed?

```suggestion
```

Reply:
It's not a duplicate, since it adds the method to roundCowState, not LogicLedger. Nonetheless, it is extraneous for now. I added while pursing a path would have made *roundCowState into a LedgerForSignature.

I'll remove.